src/frontend/src/lib/components/core/Footer.svelte

```diff
@@ -53,35 +53,44 @@
 		</div>
 
 		<div
-			class="item pointer-events-auto flex flex-row items-center justify-end gap-2 text-sm lg:max-w-48 xl:max-w-none"
+			class="item pointer-events-auto pl-6 pr-6 text-sm md:pl-0 md:pr-0 lg:max-w-48"
 			class:sm:max-w-none={$authNotSignedIn}
 			class:lg:max-w-none={$authNotSignedIn}
 			class:md:h-md:pr-4={$authNotSignedIn}
+			class:xl:max-w-none={$authNotSignedIn}
 			class:md:transition-all={$authSignedIn}
 			class:md:duration-200={$authSignedIn}
 			class:md:ease-in-out={$authSignedIn}
 			class:md:invisible={$authSignedIn}
 			class:1.5md:visible={$authSignedIn}
 			class:md:translate-x-full={$authSignedIn}
 			class:1.5md:translate-x-0={$authSignedIn}
+			class:xl:max-w-80={$authSignedIn}
+			class:1.5xl:max-w-none={$authSignedIn}
```

Comment:
With this command i say that if the screen is bigger than the defined size `1.5xl` than there is no limitation in the width anymore. 
If you are logged in it will override the `max-w-80` otherwise it will override `max-w-48`.